support node 12

Right now, `yarn` with node 12:

```
error /home/hstastna/manageiq/manageiq-ui-classic/node_modules/fs-ext: Command failed.
Exit code: 1
Command: node-gyp configure build
Arguments: 
Directory: /home/hstastna/manageiq/manageiq-ui-classic/node_modules/fs-ext
```

With react-ui-components 0.11.38, we get further... and fail on `node-sass`.

```
  g++ '-DNODE_GYP_MODULE_NAME=binding' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DBUILDING_NODE_EXTENSION' -I/home/himdel/.node-gyp/12.8.0/include/node -I/home/himdel/.node-gyp/12.8.0/src -I/home/himdel/.node-gyp/12.8.0/deps/openssl/config -I/home/himdel/.node-gyp/12.8.0/deps/openssl/openssl/include -I/home/himdel/.node-gyp/12.8.0/deps/uv/include -I/home/himdel/.node-gyp/12.8.0/deps/zlib -I/home/himdel/.node-gyp/12.8.0/deps/v8/include -I../../nan -I../src/libsass/include  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++1y -std=c++0x -MMD -MF ./Release/.deps/Release/obj.target/binding/src/create_string.o.d.raw   -c -o Release/obj.target/binding/src/create_string.o ../src/create_string.cpp
In file included from ../src/create_string.cpp:1:
../../nan/nan.h: In function ‘void Nan::AsyncQueueWorker(Nan::AsyncWorker*)’:
../../nan/nan.h:2298:62: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
                                                              ^
../src/create_string.cpp: In function ‘char* create_string(Nan::MaybeLocal<v8::Value>)’:
../src/create_string.cpp:17:37: error: no matching function for call to ‘v8::String::Utf8Value::Utf8Value(v8::Local<v8::Value>&)’
   v8::String::Utf8Value string(value);
                                     ^
In file included from /home/himdel/.node-gyp/12.8.0/include/node/node.h:63,
                 from ../../nan/nan.h:54,
                 from ../src/create_string.cpp:1:
/home/himdel/.node-gyp/12.8.0/include/node/v8.h:3002:5: note: candidate: ‘v8::String::Utf8Value::Utf8Value(v8::Isolate*, v8::Local<v8::Value>)’
     Utf8Value(Isolate* isolate, Local<v8::Value> obj);
     ^~~~~~~~~
/home/himdel/.node-gyp/12.8.0/include/node/v8.h:3002:5: note:   candidate expects 2 arguments, 1 provided
/home/himdel/.node-gyp/12.8.0/include/node/v8.h: In instantiation of ‘void v8::PersistentBase<T>::SetWeak(P*, typename v8::WeakCallbackInfo<P>::Callback, v8::WeakCallbackType) [with P = node::ObjectWrap; T = v8::Object; typename v8::WeakCallbackInfo<P>::Callback = void (*)(const v8::WeakCallbackInfo<node::ObjectWrap>&)]’:
/home/himdel/.node-gyp/12.8.0/include/node/node_object_wrap.h:84:78:   required from here
/home/himdel/.node-gyp/12.8.0/include/node/v8.h:9853:16: warning: cast between incompatible function types from ‘v8::WeakCallbackInfo<node::ObjectWrap>::Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<node::ObjectWrap>&)’} to ‘Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<void>&)’} [-Wcast-function-type]
                reinterpret_cast<Callback>(callback), type);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/himdel/.node-gyp/12.8.0/include/node/v8.h: In instantiation of ‘void v8::PersistentBase<T>::SetWeak(P*, typename v8::WeakCallbackInfo<P>::Callback, v8::WeakCallbackType) [with P = Nan::ObjectWrap; T = v8::Object; typename v8::WeakCallbackInfo<P>::Callback = void (*)(const v8::WeakCallbackInfo<Nan::ObjectWrap>&)]’:
../../nan/nan_object_wrap.h:65:61:   required from here
/home/himdel/.node-gyp/12.8.0/include/node/v8.h:9853:16: warning: cast between incompatible function types from ‘v8::WeakCallbackInfo<Nan::ObjectWrap>::Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<Nan::ObjectWrap>&)’} to ‘Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<void>&)’} [-Wcast-function-type]
make: *** [binding.target.mk:129: Release/obj.target/binding/src/create_string.o] Error 1
```

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
   "dependencies": {
     "@data-driven-forms/pf3-component-mapper": "^1.13.3",
     "@data-driven-forms/react-form-renderer": "^1.13.3",
-    "@manageiq/react-ui-components": "~0.11.37",
+    "@manageiq/react-ui-components": "~0.11.38",
     "@manageiq/ui-components": "~1.3.0",
     "@pf3/select": "~1.12.6",
     "@pf3/timeline": "~1.0.8",
@@ -131,7 +131,7 @@
     "jest-cli": "^22.0.0",
     "js-yaml": "~3.13.1",
     "ng-annotate-loader": "~0.6.1",
-    "node-sass": "~4.9.3",
+    "node-sass": "~4.12.0",
     "path-complete-extname": "~0.1.0",
     "postcss-loader": "~2.1.5",
     "postcss-smart-import": "~0.6.11",

